modules/gitops-release-notes-1-2-1.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-2-1_{context}"]
+= Release notes for {gitops-title} 1.2.1
+
+{gitops-title} 1.2.1 is now available on {product-title} 4.7, 4.8, and 4.9.
```

Comment:
GitOps 1.2.x is not available for OCP 4.9

Reply:
1.2.1 is okay on 4.9 though right? I did include the note about 1.2 not be available on 4.9 under "known issues"

Reply:
No, the first release to support OCP 4.9 will be v1.3

Reply:
I've removed 4.9 now, thanks